core-feature-pack/galleon-feature-pack/src/main/resources/packages/product.conf/content/modules/system/layers/base/org/jboss/as/product/wildfly-core/module.xml

```diff
@@ -4,8 +4,7 @@
   ~ Copyright The WildFly Authors
   ~ SPDX-License-Identifier: Apache-2.0
   -->
-
-<module xmlns="urn:jboss:module:1.9" name="org.jboss.as.product" slot="wildfly-core">
+<module xmlns="urn:jboss:module:1.5" name="org.jboss.as.product" slot="wildfly-core">
```

Comment:
Downgrading the JBoss Modules module version is unexpected here. We should keep 1.9